### ISSUE TITLE
Python

### ISSUE THREAD

**Issue by albellov** [ioannkozlitin/NumericalMethods#3]
Можно ли выполнять задания к семинарам на Python?

Белов Александр МП-30

**Comment by ioannkozlitin** [comment]
Да, Вы можете так делать. Другое дело, что это будет по-моему труднее, чем в MATLAB.
